fetch.bs

```diff
@@ -7178,6 +7174,26 @@ returns failure or a <a for=/>MIME type</a>.
 
 <hr>
 
+<div algorithm>
+<p>To <dfn id=concept-body-get-mime-type for=Body>get the MIME type</dfn>, given a {{Request}}
+or {{Response}} object, <var>requestOrResponse</var>, run these steps:
```

Comment:
```suggestion
or {{Response}} object <var>requestOrResponse</var>:
```
(no need for a comma before the variable name and no need for "run these steps" when we start with "To")